src/styles/global.css

```
@@ -0,0 +1,1023 @@
+    letter-spacing: 0.16em;
+    text-transform: uppercase;
+  }
+
+  .text-link {
+    display: inline-flex;
+    gap: 0.65rem;
+    align-items: center;
+    width: fit-content;
+    padding: 0 0 0.45rem;
+    border: 0;
+    border-bottom: 1px solid currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)